client/my-sites/stats/site.jsx

```diff
@@ -267,6 +263,39 @@ class StatsSite extends Component {
 					</div>
 				</div>
 				<JetpackColophon />
+			</>
+		);
+	}
+
+	renderEnableStatsModule() {
+		const { slug } = this.props;
+		return (
+			<EmptyContent
+				illustration="/calypso/images/illustrations/illustration-404.svg"
+				title={ translate( 'Looking for Stats?' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'Looking for stats?' )` *ES Score: 9*

Reply:
Let's reuse this case sentence string.

Reply:
Thanks, updated in https://github.com/Automattic/wp-calypso/pull/39004/commits/6fc7d4ba08e31551ffe85dc4e52cea127a5e0b9e!